engine/userdata/cloud-init/src/main/java/org/apache/cloudstack/userdata/CloudInitUserDataProvider.java

```diff
@@ -0,0 +1,208 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@vishesh92 something wrong with the rebase? This and some other files shouldn't be a part of this PR

Reply:
Yes. I messed up. Fixing it.